x-pack/plugins/cases/public/components/user_profiles/display_name.ts

```diff
@@ -6,12 +6,26 @@
  */
 
 import { UserProfileUserInfo } from '@kbn/user-profile-components/target_types/user_profile';
+import { isEmpty } from 'lodash';
 import * as i18n from './translations';
 
 export const getName = (user?: UserProfileUserInfo): string => {
   if (!user) {
     return i18n.UNKNOWN;
   }
 
-  return user.full_name ?? user.email ?? user.username;
+  return (
```

Comment:
Can we use the exposed `getUserDisplayName` function from the security plugin for consistency? (https://github.com/elastic/kibana/pull/139091)